lib/query/querybuilder.js

```diff
@@ -148,6 +148,25 @@ class Builder extends EventEmitter {
     return this;
   }
 
+  // Adds a one or more optimizer hits to the list of "optimizerHints" on the query.
+  optimizerHint() {
+    const hints = helpers.normalizeArr.apply(null, arguments);
+    if (hints.some((hint) => !isString(hint))) {
+      throw new Error('Optimizer hint must be a string');
+    }
+    if (hints.some((hint) => hint.includes('/*') || hint.includes('*/'))) {
```

Comment:
Fastest way to do this check is `indexOf <...> !== -1`

Reply:
@kibertoad I followed this review comment in https://github.com/knex/knex/pull/2815#discussion_r268410986

Reply:
fair enough :)